### DISCUSSION TITLE
Is there support for "push" API?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#9874] [Archive]
Is there support for "push" API?

**Comment by rijkvanzanten** [comment]
"push" API?

**Reply by biapar** [reply]
Yes...I know it's possibile to add with a custom code and using socket.io, I can subscribe to an API to get data without to request it.

**Reply by rijkvanzanten** [reply]
Ah you mean a native Websockets endpoint that relies on a socket connection to allow for pub-sub communication. That wasn't clear by just saying "push API" 👍🏻
